code/src/components/PostMessage.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import { HAPPY_THOUGHTS_API } from '../reusable/urls'
+
+const PostMessage = ({ messageNew, setMessageNew, messageList, setMessageList, chars, setChars }) => {
```

Comment:
messageList - do you need this in this component? I don't see you using it hehe 🧐🙂